data/lisc-ttm-sample-data.sql

```diff
@@ -146,7 +146,7 @@ PRIMARY KEY (`Cohort_ID`),
 LOCK TABLES `Cohorts` WRITE;
 /*!40000 ALTER TABLE `Cohorts` DISABLE KEYS */;
 INSERT INTO `Cohorts` VALUES (1,'TestCohort'),(2, 'AnotherCohort');
-/*!40000 ALTER TABLE `Colleges` ENABLE KEYS */;
+/*!40000 ALTER TABLE `Cohorts` ENABLE KEYS */;

```

Comment:
Was this an unrelated typo fix?

Reply:
It's in its own commit.  Without it, you can't load up the this file.

Reply:
Oh, my apologies @frankduncan.  One of the things I find challenging about GitHub is that you can view the commits as a per-commit list, or you can view the entire overall diff, but what you **can't** do is exactly the thing I want to do: see a page with the individual commits metadata at the top (author, date, commit ID, list of changed files), and *then* a single overall diff below that covering all of those commits.